### PR TITLE
mgr: misc minor changes

### PR DESCRIPTION
**Pull request by xiexingguo**

### CODE REVIEW

**Comment by xiexingguo** [comment]
@jcsp Can you take a look?(Just some cleanups, simple and easy)

Thanks!

**Comment by xiexingguo** [comment]
retest this please

File: src/mgr/MgrStandby.cc [hunk 1]
```diff
@@ -112,8 +112,7 @@ void MgrStandby::send_beacon()
                                  available);
                                  
   monc->send_mon_message(m);
-  // TODO configure period
-  timer.add_event_after(5, new C_StdFunction(
+  timer.add_event_after(g_conf->mgr_beacon_period, new C_StdFunction(
```
Comment (jcsp) [inline src/mgr/MgrStandby.cc]: Please could you add "Fixes: http://tracker.ceph.com/issues/17455" to the commit message

Comment (xiexingguo) [inline src/mgr/MgrStandby.cc]: @jcsp Done. Thanks for the review.

**Comment by xiexingguo** [comment]
@jcsp Ping

**Pull request merged**